### PR TITLE
RHDEVDOCS-6268: Update code samples

### PR DESCRIPTION
**Pull request by shivanisathe25**
Version(s): build-docs-1.1, build-docs-1.2

Issue: https://issues.redhat.com/browse/RHDEVDOCS-6268

Link to docs preview: https://85322--ocpdocs-pr.netlify.app/openshift-builds/latest/work_with_builds/using-builds.html

QE review: @avinal 
- [X] No QE for this product just yet.

Additional information:

### CODE REVIEW

File: modules/ob-creating-a-s2i-build.adoc [hunk 1]
```diff
@@ -30,9 +30,10 @@ metadata:
 spec:
   source: <1>
     type: Git
-    git:
-      url: https://github.com/shipwright-io/sample-nodejs
-    contextDir: source-build/
+    type: Git
+        git:
+          url: https://github.com/redhat-openshift-builds/samples/
```
Comment (avinal) [inline modules/ob-creating-a-s2i-build.adoc:35]:
```suggestion
          url: https://github.com/redhat-openshift-builds/samples
```

**Review by avinal — APPROVED** [review]
The extra `/` is acceptable and not at all wrong. I am just being a little paranoid. Please don't hesitate to ignore. 😵‍💫

File: modules/ob-creating-a-buildah-build.adoc [hunk 2]
```diff
@@ -97,9 +97,9 @@ EOF
 .Example: Using `shp` CLI
 [source,terminal]
 ----
-$ shp build run buildah-golang-build --follow <1>
+$ shp build run buildah-golang-build --follow
```
Comment (michaelryanpeter) [inline modules/ob-creating-a-buildah-build.adoc:100]: You had it right originally. You should use the code callout to describe the optional flag.

File: modules/ob-creating-a-s2i-build.adoc [hunk 3]
```diff
@@ -101,9 +102,9 @@ EOF
 .Example: Using `shp` CLI
 [source,terminal]
 ----
-$ shp build run s2i-nodejs-build --follow <1>
+$ shp build run s2i-nodejs-build --follow
 ----
-<1> Optional: By using the `--follow` flag, you can view the build logs in the output result.
```
Comment (michaelryanpeter) [inline modules/ob-creating-a-s2i-build.adoc:106]: See the comment above. A code callout is the way to describe the flag here. Also, you specified that flag is optional above. That makes sense. I recommend keeping it optional here for consistency.

File: modules/ob-creating-a-s2i-build.adoc [hunk 4]
```diff
@@ -101,9 +102,9 @@ EOF
...
+By using the `--follow` flag, you can view the build logs in the output result.
 
 . Check if the `BuildRun` resource is created by using one of the CLIs:
```
Comment (michaelryanpeter) [inline modules/ob-creating-a-s2i-build.adoc:109]: This is out-of-scope for this review, but it is worth fixing in the future. When you do fix this, it's best to sweep your documentation set and fix all the places where this is used.

```suggestion
. Check if the `BuildRun` resource is created by running one of the following commands:
```

**Review by michaelryanpeter — COMMENTED** [review]
One small issue, repeated, and a terminology issue that should be fixed in the future, in a follow up issue.

Also, please remember to provide a direct link to the issue in comment zero of the PR.

/label peer-review-done
/remove-label peer-review-needed
/remove-label peer-review-in-progress

**Comment by dfitzmau** [comment]
Hi @shivanisathe25 . A QE review is needed before I can merge this PR.

**Comment by shivanisathe25** [comment]
> Hi @shivanisathe25 . I've finished my review. A QE review is needed before I can merge this PR. Also, does a Jira exist for this issue? I do not see any mentioned in the Description box.

Hi, the team doesnt have a QE. Only the SMEs approve the PRs for now.

**Pull request merged**